src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine array1 = new Bulldozer();
+        Machine array2 = new Excavator();
+        Machine array3 = new Truck();
```

Comment:
```suggestion
        Machine bulldozer = new Bulldozer();
        Machine excavator = new Excavator();
        Machine truck = new Truck();
        
        Machine[] machines = new Machine[]{bulldozer, excavator, truck};
```
fix naming
Also it would be really good to put machines in an array and than call for-loop to invoke methods on them

Reply:
![image](https://user-images.githubusercontent.com/83697132/184209567-f7c8dc88-2d23-4f0a-8b84-903d0edbc19a.png)
I need some help with for loop)

Reply:
Don`t understand from which array I must activate loop